Set/Game/GameView.swift

```diff
@@ -9,9 +9,16 @@ struct GameView: View {
             AspectVGrid(items: gameViewModel.cardsOnScreen, aspectRatio: 2 / 3) { card in
                 cardViewBuilder.build(for: card)
                     .onTapGesture {
-                        gameViewModel.select(card)
+                        guard let player = gameViewModel.whoseTurn else {
+                            return
+                        }
+                        gameViewModel.select(card, player)
                     }
             }
+            .onReceive(gameViewModel.timer) { _ in
+                gameViewModel.startTimer()
```

Comment:
I applied incorrect name therefore it confused you.
We listen timer on line 18 and perform an action. 
You can take a look at good example here - https://youtu.be/ymXRX6ZB-J0?t=608

To not implement the action performance in the View, I decided to move in to the ViewModel, but maybe it would be better to call it "PerformCountingDownAction" instead of "StartTimer"

Reply:
fixed